schemas/gen.py

```diff
@@ -299,22 +301,48 @@ def go_getters(struct: str, schema: Schema, spec: List[FieldSpec]) -> List[str]:
     for field, type, tag in spec:
         defaulted_type, default, required = get_defaulted_type(schema, tag, type)
 
+        if not field.startswith("Raw"):
+            raise AssertionError(
+                f'{struct} has field {field} which doesn\'t start with "Raw"; all fields should '
+                'start with "Raw" and the getter will be the primary API for accessing those '
+                "values.  When the field is a pointer-type with a non-nil default type, the getter "
+                "will be a non-pointer (automatic dereferencing) for use after WithDefaults() is "
+                "used to fill the default values."
+            )
+
+        getter = field[len("Raw") :]
+
         if defaulted_type == type:
-            lines.append(f"func ({x} {struct}) Get{field}() {type} {{")
+            # Getter for nonpointer field.
+            lines.append(f"func ({x} {struct}) {getter}() {type} {{")
             lines.append(f"\treturn {x}.{field}")
             lines.append("}")
             lines.append("")
+
+            # Setter for nonpointer field.
+            lines.append(f"func ({x} {struct}) Set{getter}(val {type}) {{")
```

Comment:
Should this generate a setter with a pointer receiver? Seems like most of these setters won't work.
```suggestion
            lines.append(f"func ({x} *{struct}) Set{getter}(val {type}) {{")
```

Reply:
Ah, yes!  Great catch.